src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -17,6 +17,8 @@ gregtech.top.transform_up=§cStep Up§r %s (%dA) -> %s (%dA)
 gregtech.top.transform_down=§aStep Down§r %s (%dA) -> %s (%dA)
 gregtech.top.transform_input=§6Input:§r %s (%dA)
 gregtech.top.transform_output=§9Output:§r %s (%dA)
+gregtech.top.fuel_info=/%,d %,d s
```

Comment:
Wouldn't it be better to add a little bit bigger space?
```suggestion
gregtech.top.fuel_info=/%,d\t%,d s
```
Also I think it would print these two in a column but not sure. Can you provide screenshots for first two with `\t` (tab character). I really don't like how space looks in first two screenshots, I think it can be easily fixed.